pkg/api/update/update.go

```diff
@@ -93,7 +93,8 @@ func (handle *Handler) Handle(w http.ResponseWriter, r *http.Request) {
 
 func handleError(w http.ResponseWriter, err error) {
 	if err != nil {
-		if errors.Is(err, &types.ValidationError{}) {
```

Comment:
`errors.Is` was only working with sentinel errors (e.g errors created with `errors.New`) so it was failing in this case.